packages/react/package.json

```diff
@@ -10,14 +10,15 @@
   },
   "version": "0.28.36",
   "main": "lib/hig-react.js",
-  "files": ["lib/*"],
+  "files": [
+    "lib/*"
+  ],
   "scripts": {
     "babel": "babel --ignore test.js,Tests.js,test -d lib src",
     "build-playground": "react-scripts build",
     "build-icons": "node scripts/build-icons.js",
     "deploy-playground": "react-scripts build && surge build",
-    "gemini":
-      "concurrently 'yarn storybook' 'gemini test --reporter html --reporter flat' --kill-others --success first",
```

Comment:
Observation: I think ESLint keeps moving this and the `lib/*` line to their own line, but yarn moves it back. Want to see if adding `package.json` to `.eslintignore` helps?

Reply:
yeah. I have seen this too.

Reply:
Just ran the script and no changes. This may also be caused by editors with "format on save" enabled. Nevertheless, yarn is the formatting we want, right? So lets figure that out in another PR.

Reply:
Good call, this was my Beautify extension. Carry on :)